pom.xml

```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.folio</groupId>
   <artifactId>edge-rtac</artifactId>
-  <version>2.6.0-SNAPSHOT</version>
+  <version>2.5.2-SNAPSHOT</version>
```

Comment:
This seems like an odd jump down, is this because the pom changes were cherry picked from the mainline?

I would expect the base of the b2.5 branch to have the version as `2.5.0`

Reply:
The HEAD of the b2.5 branch https://github.com/folio-org/edge-rtac/commits/b2.5
incorrectly has `<version>2.6.0-SNAPSHOT</version>` (the same as master branch).
It should have `<version>2.5.1-SNAPSHOT</version>`.

When this release PR gets merged the HEAD of b2.5 will have the correct version.

Reply:
Thank you for correcting this.

Do you know why the b2.5 branch got the incorrect version in the first place? 

As this is the first patch release, I wouldn't have expected it to exist until during this work.